homeassistant/components/islamic_prayer_times/config_flow.py

```diff
@@ -1,10 +1,13 @@
 """Config flow for Islamic Prayer Times integration."""
 from __future__ import annotations
 
+from typing import Any
+
```

Comment:
These are just type hint improvements.
Is there any chance you could create a separate PR for "Improve type hints in islamic_prayer_times"?

I also feel that it would be easier to review if there was a separate PR for "migration to EntityDescription" and "migration to Coordinator"

Reply:
This hasn't been addressed

Reply:
Breaking up the PR would require rewriting parts of the code (especially tests) to get it working and pass the tests. Appreciate approving this PR as it is.

Reply:
But I will defer to Martin...

Reply:
It's ok. There's just a small thing left to address before we can merge.